From 05f3f1c93ea5a204ab96aeea7ca5319d0e1f6a7c Mon Sep 17 00:00:00 2001
From: Julian Wollrath <jwollrath@web.de>
Date: Thu, 18 May 2017 14:24:04 +0200
Subject: [PATCH 1/4] Provide notification-daemon (Closes: #734645)

With naughty, a notification daemon is provided. Make this clear by
putting it into the provides field on debian/control.
---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index a08dd57b..d57cf9de 100644
--- a/debian/control
+++ b/debian/control
@@ -49,7 +49,7 @@ Depends: dbus-x11,
          menu,
          ${misc:Depends},
          ${shlibs:Depends}
-Provides: x-window-manager
+Provides: notification-daemon, x-window-manager
 Description: highly configurable X window manager
  awesome manages windows dynamically in floating or tiled layouts. It is
  primarily targeted at power users, developers, and all those dealing with
-- 
2.13.2

